editor/inspector/assets/fbx/preview.js

```diff
@@ -339,7 +339,15 @@ const Elements = {
         ready() {
             const panel = this;
             panel.$.canvas.addEventListener('mousedown', async (event) => {
-                await callModelPreviewFunction('onMouseDown', { x: event.x, y: event.y });
+                await callModelPreviewFunction('onMouseDown', { x: event.x, y: event.y, button: event.button });
+
+                async function keydown(event) {
+                    await callModelPreviewFunction('onKeyDown', {
```

Comment:
1. Shouldn't a generic keydown event be triggered after mousedown?
2. If it's an event on the document, should we still use 'onKeyDown'? What should we name it if we need to listen to events on the canvas in the future?